src/swapnaWJun20/PalindromeNo.java

```diff
@@ -0,0 +1,25 @@
+package swapnaWJun20;
+
+public class PalindromeNo {
+
+	void findPalindromeNumber(int number) {
+		int number1 = number;
+		int temp = 0;
+		int mod = 0;
+		while (number1 > 0) {
+			mod = number1 % 10;
+			number1 = number1 / 10;
+			temp = mod + temp * 10;
+		}
```

Comment:
Logic is correct but the code can be optimized and the mod variable is not required. instead, you can write in the below way. Please remove the commented line below and  update the logic you will get the result.

//int mod = 0;
		while (number1 > 0) {
			temp = temp * 10 + number1 % 10;
			number1 = number1 / 10;
			//temp = mod + temp * 10;
		}

Reply:
Above mentioned changes have been incorporated in the code.

Reply:
Now it looks good. For the math operation, it would be good to separate these by brackets. 
temp = (temp * 10 )+ (number1 % 10);